docs/guide/Core_Component.md

```diff
@@ -13,6 +13,13 @@ _Includes: Ballot Encryption_
 
 :fontawesome-brands-github: Source - ✨ Coming Soon
 
+## Java
```

Comment:
Can we actually make this:
```
## Community

### Java
```

Reply:
This way it can be highlighted that it's done by the community.

Reply:
done